### PR TITLE
fix(changelog): Add new line between changes

### PR DESCRIPTION
**Pull request by shrujalshah28**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by shrujalshah28** [comment]
@googlebot I signed it!

**Comment by atscott** [comment]
Hi @shrujalshah28, thanks for correcting this mistake. Could you update the commit message to be "docs: ..." instead of "fix..." (reserved for bug fixes). Can you also create a second PR that would fix this on the `9.1.x` branch?

**Review by atscott — APPROVED** [review]

**Comment by shrujalshah28** [comment]
I have updated the commit message.
And I checked `9.1.x` branch but that brach doesn't have this issue.

**Review by gkalpak — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +36,9 @@
...
+* **router:** pass correct component to canDeactivate checks when using two or more sibling router-outlets ([#36302](https://github.com/angular/angular/issues/36302)) ([8e7f903](https://github.com/angular/angular/commit/8e7f903)), closes [#34614](https://github.com/angular/angular/issues/34614)
+* **upgrade:** update $locationShim to handle Location changes before initialization ([#36498](https://github.com/angular/angular/issues/36498)) ([a67afcc](https://github.com/angular/angular/commit/a67afcc)), closes [#36492](https://github.com/angular/angular/issues/36492)
 
 ### Performance Improvements
 * **ngcc:** only load  if it is needed ([#36486](https://github.com/angular/angular/issues/36486)) ([e06512b](https://github.com/angular/angular/commit/e06512b))                                                                             * **ngcc:** read dependencies from entry-point manifest ([#36486](https://github.com/angular/angular/issues/36486)) ([918e628](https://github.com/angular/angular/commit/918e628)), closes [#issuecomment-608401834](https://github.com/angular/angular/issues/issuecomment-608401834)
```
Comment (BO41) [inline CHANGELOG.md:44]: Same here ;)

**Pull request closed without merging**